move_base_msgs/msg/RecoveryStatus.msg

```diff
@@ -0,0 +1,4 @@
+geometry_msgs/PoseStamped pose_stamped
+int16 index
+int16 size
+string name
```

Comment:
At first blush, just from the names of the variables here, I don't know what this message is supposed to do.

Reply:
I was using the name to log which recovery behaviors were executing, the index/size to indicate where in the list of recovery behaviors I currently was, and pose to figure out pose_stamped so I can plot failures.

EX:
https://github.com/PeterMitrano/recovery_supervisor/blob/master/recovery_supervisor/src/recovery_supervisor.cpp